www/themes/Gamma/templates/frontend/profile.tpl

```diff
@@ -54,7 +58,7 @@
 		<th>Grabs Total:</th>
 		<td>{$user.grabs}</td>
 	</tr>
-	{if ($user.id==$userdata.id || $userdata.role==2) && $site->registerstatus==1}
+	{if (!$publicview || $isadmin) && $site->registerstatus==1}
```

Comment:
https://github.com/nZEDb/nZEDb/blob/master/www/themes/alpha/templates/frontend/profile.tpl#L44

It is same as in alpha.